cmd/lakectl/cmd/config/config.go

```diff
@@ -64,11 +79,16 @@ const (
 	HiveDBLocationURIKey = "metastore.hive.db_location_uri"
 
 	// Defaults
-	HiveDBLocationURI = "file:/user/hive/warehouse/"
+	HiveDBLocationURI        = "file:/user/hive/warehouse/"
+	ConfigServerEndpointURL  = "server.endpoint_url"
+	DefaultServerEndpointURL = "http://127.0.0.1:8000"
+	ConfigAccessKeyID        = "credentials.access_key_id"
+	ConfigSecretAccessKey    = "credentials.secret_access_key"
```

Comment:
These are key names so presumably if anything gets a "Key" suffix then these should too.

Again, I don't mind either way, just please let's be consistent.